sdk/src/MemberAsync.cs

```diff
@@ -534,7 +534,7 @@ public Task<Blob> GetProfilePicture(string memberId, ProfilePictureSize size)
         /// <param name="tokenRequest">the token request</param>
         /// <returns>an id to reference the token request</returns>
         [Obsolete("Deprecated. Use StoreTokenRequest(TokenRequestPayload, TokenRequestOptions) instead.")]
-        public Task<string> StoreTokenRequest(TokenRequest tokenRequest)
+        public Task<string> StoreTokenRequest(Proto.Common.TokenProtos.TokenRequest tokenRequest)
```

Comment:
import? or does the new sdk object `TokenRequest` creates a conflict somehow?

Reply:
Yes, the IDE was forcing me to have full name because of the conflict.